cdbot/cogs/fun.py

```diff
@@ -151,6 +151,10 @@ def check(reaction, user):
         if "revive" in message.content.lower():
             await emojify(message, "nou")
 
+        # Adds waving emoji when a new user joins.
+        if "Welcome to the Cyber Discovery" in message.content and message.author.id == WELCOME_BOT_ID
+            await message.add_reaction("👋")
```

Comment:
So a later pull request has now been merged in, #157, which uses a new syntax for referencing emoji characters. Instead of including the literal character, which can be difficult to view in some text editors, we now refer to non-ASCII characters by *name*.

That is to say
```suggestion
            await message.add_reaction("\N{THUMBS UP SIGN}")
```
Could you make that change here so as to keep the codebase consistent?

Reply:
What jokebook said